groups/xml-infrastructure/templates/bep_cron.tpl

```diff
@@ -1,93 +1,108 @@
-#########################LIVE SERVICE PROCESSING START########################
-#########################Form Partition#######################################
-# Monday to Tuesday
-*/1 * * * 1-2 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
-#*/1 0-19 * * 1-2 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
-#*/1 22-23 * * 1-2 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE#########################LIVE SERVICE PROCESSING START########################
+#LIVE#########################Form Partition#######################################
+#LIVE# Monday to Tuesday
+#LIVE*/1 * * * 1-2 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE#*/1 0-19 * * 1-2 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE#*/1 22-23 * * 1-2 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE
+#LIVE# Wednesday
+#LIVE*/1 * * * 3 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE#*/1 0-18 * * 3 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE#*/1 22-23 * * 3 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE
+#LIVE# Thursday
+#LIVE*/1 * * * 4 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE#*/1 0-18 * * 4 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE#*/1 22-23 * * 4 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE
+#LIVE# Friday
+#LIVE*/1 * * * 5 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE#*/1 0-18 * * 5 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE#*/1 22-23 * * 5 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE
+#LIVE# Saturday
+#LIVE*/1 0-23 * * 6 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE#*/1 0-6 * * 6 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE
+#LIVE# Sunday
+#LIVE*/1 0-23 * * 0 /home/xml/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE#*/1 0-18 * * 0 /home/ewfbeplive/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE#*/1 20-23 * * 0 /home/ewfbeplive/efbackend/formPartition.sh >/dev/null 2>&1
```

Comment:
Just noticed the home directory is incorrect here. Should be `/home/xml`.

Reply:
just corrected